src/xGadgetron/cGadgetron/include/sirf/Gadgetron/encoding.h

```diff
@@ -0,0 +1,156 @@
+/*
+SyneRBI Synergistic Image Reconstruction Framework (SIRF)
+Copyright 2015 - 2021 Rutherford Appleton Laboratory STFC
+
+This is software developed for the Collaborative Computational
+Project in Synergistic Reconstruction for Biomedical Imaging (formerly CCP PETMR)
+(http://www.ccpsynerbi.ac.uk/).
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+/*!
+\file
+\ingroup Fourier encoding
+\brief File for cartesian fourier encoding.
+
+\author Johannes Mayer
+\author SyneRBI
+*/
+
+#ifndef ENCODING_H
+#define ENCODING_H
+
+#include <vector>
+
+#include <ismrmrd/xml.h>
+#include "sirf/Gadgetron/gadgetron_data_containers.h"
+//#include "sirf/Gadgetron/gadgetron_image_wrap.h"
+#include "sirf/iUtilities/LocalisedException.h"
+
+#include <ismrmrd/ismrmrd.h>
+
+/*!
+\file
+\ingroup Fourier Encoding
+\brief Specification file for preparing MRAcquisitionData for Fourier encoding.
```

Comment:
```suggestion
\brief Specification file for preparing sirf.MRAcquisitionData for Fourier encoding.
```
needs namespace as comment is outside it